api/db_queries.py

```diff
@@ -37,3 +37,33 @@
             RETURN { [category["category"]]: category.categoryname }
 )
 """
+
+query_items_for_menu = """
```

Comment:
naming: even though this is currently used for the menu, I would avoid naming database queries after specific UI elements. IMHO a better name would describe what the query does, like `query_files_for_category` (just an example)